awspub/tests/test_context_validation.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import pathlib
+from pydantic import ValidationError
+from awspub.context import Context
+from unittest.mock import patch, mock_open
+
+
+def test_valid_configuration():
+    with patch('awspub.context.pathlib.Path.is_absolute', return_value=True):
```

Comment:
as part of moving these to `test_context.py` We should prefer to use sample files rather than inline dictionaries (this will also negate the need to mock out `open`)